eslint.config.mjs

```diff
@@ -0,0 +1,8 @@
+import CodeX from '../eslint-config/index.js';
```

Comment:
Where does this `eslint-config/index.js` lie? 🤔

Reply:
![image](https://github.com/codex-team/notes.api/assets/31101125/76768db1-7113-48c9-a601-ed915025188c)

Reply:
this is for tests, i have cloned eslint-config as repo and it is in one folder with notes.api, so i can change codex eslint-config locally

Reply:
changed this import, now imported from downloaded package